src/porepy/grids/refinement.py

```diff
@@ -285,6 +288,232 @@ def remesh_1d(g_old: pp.Grid, num_nodes: int, tol: float = 1e-6) -> pp.Grid:
     return g
 
 
+def mdg_refinement(
+    mdg: pp.MixedDimensionalGrid,
+    mdg_ref: pp.MixedDimensionalGrid,
+    tol: float = 1e-8,
+    mode: Literal["nested"] = "nested",
```

Comment:
Does this argument have a long-term utility?

Reply:
No idea!

Reply:
I suggest we keep it for now. The pair of setting this up again will be substantial, and the code bothers nobody at the moment.